index.js

```diff
@@ -32,15 +32,49 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("h1#header")
 function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+  if (element.target.style.color === "red") {
+    element.target.style.color = "black"
   } else {
-    element.style.color = "red"
+    element.target.style.color = "red"
   }
 }
-
+header.addEventListener("click", toggleColor)
 
 /***** Deliverable 2 *****/
+const addPlayerForm = document.querySelector("#new-player-form")
+
+function addPlayerSubmit(event) {
+  event.preventDefault()
+  const playerForm = event.target
+  
+  const number = playerForm.number.value
+  const name = playerForm.name.value
+  const nickname = playerForm.nickname.value
+  const photo = playerForm.photo.value
+
+  playerObj = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 1000
+  }
```

Comment:
Careful with the variable declaration here (make sure to use `const`/`let` in front of `playerObj`)

Also, if you want, you can shorten this up a bit by declaring the object with the values directly:

```js
const playerObj = {
    number: playerForm.number.value,
    name: playerForm.name.value,
    nickname: playerForm.nickname.value,
    photo: playerForm.photo.value,
    likes: 1000
}
```